src/Propel/Generator/Command/Helper/DialogHelper3.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Propel\Generator\Command\Helper;
+
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class DialogHelper3 extends QuestionHelper implements DialogHelperInterface
+{
+    /** @var InputInterface $input */
+    protected $input;
+
+    /** @var OutputInterface $output */
+    protected $output;
+
+    public function __construct(InputInterface $input, OutputInterface $output)
+    {
+        $this->input = $input;
+        $this->output = $output;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function askQuestion($question, $default = null, array $autocomplete = null)
+    {
+        $question = new Question($this->formatQuestion($question, $default), $default);
+        $question->setAutocompleterValues($autocomplete);
+
+        return parent::ask($this->input, $this->output, $question);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function askHiddenResponse($question, $fallback = true)
+    {
+        $question = new Question($question);
```

Comment:
There's a `$this->formatQuestion` missing here. It should be `new Question($this->formatQuestion($question))` :) Otherwise the question is not followed by a `:` and also has no color.